config/mapconverter/expandmapconverter/expand_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/collector/config"
-	"go.opentelemetry.io/collector/config/mapprovider/filemapprovider"
+	"go.opentelemetry.io/collector/config/configtest"
```

Comment:
Changes in this file are extracted as https://github.com/open-telemetry/opentelemetry-collector/pull/5229 which will be merged first.